c2corg_ui/templates/helpers/view.html

```diff
@@ -216,7 +224,11 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
           <app-card app-card-doc="doc"></app-card>
           % if type == 'articles':
             <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+              ng-if="userCtrl.auth.isAuthenticated()
+              % if isCollaborative:
```

Comment:
Why not simply write
```
% if is_collaborative(document):
```
?
I don't understand why you have introduced the ``isCollaborative`` variable? Ditto for the other changes below.

To avoid multilines in the ``ng-if`` you could write instead ``${'&& userCtrl.auth.isModerator()' if is_collaborative(document) else ''}``.